app/views/posts/index.html.erb

```diff
@@ -0,0 +1,57 @@
+<h1>Users Post list</h1>
+
+<main id="posts" class="page">
+   <div id="user-page">
+    <div class="btn-container">
+      <div class="home-link-container ">
+      <%= link_to "Home", root_path, class: "home-link2" %>
+      </div>
+      <div class="back-link-container ">
+        <%= link_to "Back", :back, class: "back-link2" %> 
+      </div> 
+    </div>
+  </div>
+
+    <div class=" user-item">
+       <div class="user-image">
+ <%= image_tag('user.png', width: "150px", height: "100px") %>
+ </div>
+ <div class="user-details">
+      <h2> <%= link_to @user.name, user_path(@user) %></h2>
+      <p class="user-posts-counter">Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
```

Comment:
- [Optional] Check comment above and keep codebase DRY

Reply:
implemented